weaviate/collections/aggregations/hybrid.py

```diff
@@ -25,6 +25,7 @@ async def hybrid(
         filters: Optional[_Filters] = None,
         group_by: Optional[Union[str, GroupByAggregate]] = None,
         target_vector: Optional[str] = None,
+        max_vector_distance: Optional[float] = None,
```

Comment:
I noticed that this is `Optional[NUMBER]` in the `.queries` methods. Should we align these or is this a point-of-difference due to the differing under-the-hood protocols?